src/sentry/web/urls.py

```diff
@@ -481,6 +481,12 @@
         react_page_view,
         name="stories",
     ),
+    # Notification Previews
+    re_path(
+        r"^notifs/",
```

Comment:
is there a better name we can give this route?

Reply:
oh i guess we've called the whole thing 'notifs'

Reply:
Yeah I agree. Having a common route for dev debug tools would be nice, and gives us a way to conditionally disable them in prod if we decide to.